container/Dockerfile.gpu

```diff
@@ -60,11 +60,12 @@ RUN apt-get update && \
     openjdk-8-jdk-headless \
     python3-dev \
     gcc \
+    g++ \
```

Comment:
I fixed CI `pip3` issues in https://github.com/neo-ai/neo-ai-dlr/pull/164. `g++` is not needed now.

Reply:
@apivovarov when can you merge your changes for the CI issues ? This currently blocks inference containers CI/CD deployment to production as the inference containers build is broken on mainline?

Reply:
Since it is urgent I moved my changes to a separate small PR https://github.com/neo-ai/neo-ai-dlr/pull/165. I also included MMS version removal to my PR